internal/cloudapi/v1/v1.go

```diff
@@ -175,9 +175,9 @@ func (h *apiHandlers) Compose(ctx echo.Context) error {
 
 		if depsolveResults.Error != "" {
 			if depsolveResults.ErrorType == worker.DepsolveErrorType {
-				return echo.NewHTTPError(http.StatusBadRequest, "Failed to depsolve requested package set: %s", depsolveResults.Error)
+				return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Failed to depsolve requested package set: %s", depsolveResults.Error))
```

Comment:
Why add `fmt.Sprintf` ? Also cloudapi/v1 is deprecated (everything uses v2 now). It's also untested. I'd be ok with dropping any changes to cloudapi/v1 (in fact we sohuld just remove cloudapi/v1 soon, separate PR of course).

Reply:
Agreed.

Reply:
Oh yeah.. I really have to stop using v1 for my tests!